docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc

```diff
@@ -0,0 +1,214 @@
+= OneCX PortalDialogService
+
+:dynamic_dialog_config_url: https://primeng.org/dynamicdialog#api.dynamicdialog-config
+
+== Overview
+This document aims to showcase how to manage dialogs using OneCX libraries.
+
+== Importing PortalDialogService
+To properly use PortalDialogService, please import it via the `providePortalDialogService`.
+
+.Provide PortalDialogService via the providers array
+[source,javascript]
+----
+import {
+  providePortalDialogService
+} from '@onecx/portal-integration-angular';
+
+...
+providers: [
+    providePortalDialogService(),
+]
+...
+----
```

Comment:
maybe note this has to be done in the module? Or in which file it needs to be respectively (e.g., standalone?)